### PR TITLE
Add build stableFdroid CI

### PR DESCRIPTION
**Pull request by mimi89999**

### CODE REVIEW

**Review by bassosimone — COMMENTED** [review]
Thanks for starting this pull request! Two questions:

1. would adding a `connectedStableFdroidDebugAndroidTest` to `emulator.yml` be useful to check whether we're breaking the f-droid build? https://github.com/ooni/probe-android/blob/master/.github/workflows/emulator.yml

2. in which environment does the f-droid build run? Is it a Docker container perhaps? How can we replicate such an environment here (including extra scripts) such that we can really be confident that a future build will successfully complete in f-droid?

Thanks again! 🚀

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+name: build
+on:
+  push:
```
Comment (bassosimone) [inline .github/workflows/build.yml:3]:
```suggestion
  push:
  pull_request:
```

**Comment by lorenzoPrimi** [comment]
@bassosimone can this be closed or merged?

**Comment by bassosimone** [comment]
Superseded by https://github.com/ooni/probe-android/pull/408#pullrequestreview-578474313

**Pull request closed without merging**